packages/flutter/test/material/slider_test.dart

```diff
@@ -2473,4 +2473,43 @@ void main() {
     // 24.0 is the default margin, (800.0 - 24.0 - 24.0) is the slider's width.
     expect(nearEqual(activeTrackRRect.right, (800.0 - 24.0 - 24.0) * (5 / 15) + 24.0, 0.01), true);
   });
+
+  testWidgets('If thumbColor is null, it defaults to CupertinoColors.white',
```

Comment:
Please add more tests that test that `thumbColor` is respected when a `Slider` is created with it set, and when `Slider.adaptive` is created on `TargetPlatform.android`.

Ideally all of these tests would also verify that the thumb was actually drawn with the color, and not just that it set the attribute, which isn't really testing much, other than Dart parameter assignment in the constructor.

You can find examples of how to capture the color from the renderObject and test it elsewhere in this file. Let me know if you need help with that part.

Reply:
I don't think it has anything to do with your PR. We've been having some major issues with our Windows bots lately.

Reply:
Okay thanks a lot 👍🏽